pkg/service/catalog.go

```diff
@@ -17,7 +17,7 @@ type catalog struct {
 	Services        []json.RawMessage `json:"services"`
 	services        []Service
 	indexedServices map[string]Service
-	jsonMutex       sync.Mutex
+	jsonMutex       *sync.Mutex
```

Comment:
I made this change because when I refactored the method receivers to not be pointers i got lots of complaints about passing around the Mutex copy